fu-ilab-swp18/pflanzen1
# Humidity sensor

This adds sensor functions to read out humidity and/or light from the boards, also a stub for a sensor thread.

It also adds an `adc_test`, which is just the adapted example from watr.li.

Test plan:
* Use a samr21-xpro with a connected humidity sensor.
* Run `BOARD=samr21-xpro make term flash`.
* There should be periodic output fromt he sensor thread. Test dry and with some water.
